src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml.cs

```diff
@@ -177,9 +177,9 @@ private async void NewLayoutButton_Click(object sender, RoutedEventArgs e)
             foreach (LayoutModel customModel in MainWindowSettingsModel.CustomModels)
             {
                 string name = customModel.Name;
-                if (name.StartsWith(defaultNamePrefix))
+                if (name.StartsWith(defaultNamePrefix, StringComparison.Ordinal))
                 {
-                    if (int.TryParse(name.Substring(defaultNamePrefix.Length), out int i))
+                    if (int.TryParse(name.AsSpan(defaultNamePrefix.Length), out int i))
```

Comment:
Why span?  Something added to lang?

Reply:
It's the new hotness in net 6. This fixes [CA1832: Use AsSpan or AsMemory instead of Range-based indexers for getting ReadOnlySpan or ReadOnlyMemory portion of an array](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1832). Substring creates an new string (basically a char array) which is not needed as we don't modify it here. AsSpan returns the relevant part of the string but doesn't allocate anything.

Pretty irrelevant here but since each warning is treat as an error I had no choice but fixing this

Reply:
> Last few fixes I think @SeraphimaZ may want to glance at but most of this seems straightforward

I would like @SeraphimaZ to have a look too! FancyZones is the project which I use most. Don't want to be the one who breaks it :-)